modules/service/src/main/scala/lucuma/odb/graphql/input/AngleInput.scala

```diff
@@ -40,20 +40,6 @@ object AngleInput {
   val DMS             = StringBinding.emap(getDMS)
   val HMS             = StringBinding.emap(getHMS)
 
-
-  def decimalInputHandler: PartialFunction[(BigDecimal, String), Result[Angle]] = {
-    case (v, "MICROARCSECONDS")    => Result(getMicroarcseconds(v.toLong))
-    case (v, "MICROSECONDS")    => Result(getMicroseconds(v.toLong))
-    case (v, "MILLIARCSECONDS") => Result(getMilliarcseconds(v.toInt))
-    case (v, "MILLISECONDS")    => Result(getMilliseconds(v.toInt))
-    case (v, "ARCSECONDS")      => Result(getArcSeconds(v.toDouble))
-    case (v, "SECONDS")         => Result(getSeconds(v.toInt))
-    case (v, "ARCMINUTES")      => Result(getArcMinutes(v.toInt))
-    case (v, "MINUTES")         => Result(getMinutes(v.toInt))
-    case (v, "DEGREES")         => Result(getDegrees(v.toDouble))
-    case (v, "HOURS")           => Result(getHours(v.toDouble))
-  }
```

Comment:
This was left over from the days when the API had "decimal" and "integer" inputs but was no longer being used.